### PR TITLE
Garrison SQL Changes

### PR DESCRIPTION
**Pull request by RunAwayDSP**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

All Garrison SQL changes (so it stops shifting on me) Please merge this so I don't have to shift/update these ever again 🙏

### CODE REVIEW

File: sql/mob_pools.sql [hunk 1]
```diff
@@ -7090,6 +7090,8 @@ INSERT INTO `mob_pools` VALUES (7037,'Sapphire_Quadav_LITD','Sapphire_Quadav',20
 INSERT INTO `mob_pools` VALUES (7038,'Excaliace','Excaliace',145,0x01000503AC10A720A730A740A750006000700000,1,1,5,240,100,0,0,0,0,0,0,32,7,1179,0,0,0,64,256,0,145);
 INSERT INTO `mob_pools` VALUES (7039,'Ixzdei_RDM','Ixzdei',272,0x0000860400000000000000000000000000000000,4,4,7,240,100,0,1,1,0,2,16,0,0,1153,6,0,468,0,0,272,272);
 
+INSERT INTO `mob_pools` VALUES (7040,'Hastatus_XIII-XXV','Hastatus_XIII-XXV',25,0x0000EF0400000000000000000000000000000000,1,1,5,265,100,0,1,0,1,0,0,0,0,1155,0,0,0,0,0,25,25); -- Was missing somehow?
```
Comment (zach2good) [inline sql/mob_pools.sql:7093]: Meaningless comment for future readers

File: sql/mob_droplist.sql [hunk 2]
```diff
@@ -26777,6 +26777,10 @@ INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2815,@COMMON);  -- Aptant Of Ar
 -- ZoneID: 125 - Picolaton
 INSERT INTO `mob_droplist` VALUES (3225,0,0,1000,11343,@COMMON); -- Thrakon Breastplate (Common, 15%)
 
+-- Garrison Drops
```
Comment (zach2good) [inline sql/mob_droplist.sql:26780]: Do mobs in garrison drop anything else apart from these? I don't remember if they do or not

**Review by zach2good — COMMENTED** [review]
Good idea, sql shifting is the bane of long running work

File: sql/mob_droplist.sql (hunk 2 above)
Comment (RunAwayDSP) [reply sql/mob_droplist.sql:26780]: possibly seals/geodes but I don't even remember seeing those either

**Review by zach2good — APPROVED** [review]

**Pull request merged**